### DISCUSSION TITLE
ARM for Zimbra

### DISCUSSION THREAD

**Discussion by fufiderheld** [ianw1974/zimbra-build-scripts#102] [General]
Is there any chance that this can be used for the ubuntu ARM Version?
Why? Cause I have the turingpi Board with 4 RK1 running and it would be really awesome to run my Zimbra OSE 8.8.15 updated on my RK1. 
Any idear how i can do that?
Greetings Wolfgang

**Comment by ianw1974** [comment]
I don't know.  First, my Zimbra script builds for 9 and 10 only, not for version 8.  Also, I've no idea if Zimbra can actually be compiled and ran on arm.  I also don't have arm hardware to test and answer the question.

**Reply by fufiderheld** [reply]
Yeah I know it s for Version 9 or 10, this is how I wanted to update mine ;-) 
So thanks for the skript and the work!
I'll give it a try and yeah, lets see what will happen ;-)

**Comment by jeredfloyd** [comment]
As Zimbra only supports x86_64, it is unlikely that an aarch64 build will be successful.  You can certainly try to use the build scripts and see what blockers arise!

**Reply by fufiderheld** [reply]
Thanks, I was wondering if someone ever did that. But I wasn't really lucky in finding anything with Zimbra AND Arm.
